src/neuroconv/datainterfaces/ecephys/kilosort/kilosortdatainterface.py

```diff
@@ -36,3 +36,31 @@ def __init__(
         verbose: bool, default: True
         """
         super().__init__(folder_path=folder_path, keep_good_only=keep_good_only, verbose=verbose)
+
+    def get_metadata(self):
+        metadata = super().get_metadata()
+        # See Kilosort save_to_phy() docstring for more info on these fields: https://github.com/MouseLand/Kilosort/blob/main/kilosort/io.py
+        # Or see phy documentation: https://github.com/cortex-lab/phy/blob/master/phy/apps/base.py
+        metadata["Ecephys"]["UnitProperties"] = [
+            dict(name="units", description="Autogenerated by neuroconv."),
```

Comment:
Wait, what is this one? Is it for the name of the units table?

Reply:
For some reason, I thought I needed to include this, but it was an artifact from how I was reading the UnitProperties for Schneider Conversion.